### PR TITLE
Add contributing guide.

### PR DESCRIPTION
**Pull request by ilopezgp**
This PR adds a contributing guide to the documentation, based on the guide in use for ClimateMachine.

### CODE REVIEW

File: docs/src/contributing.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+are not necessary will be skipped.  For example doc only changes
+do not require the unit tests to be run.
+
+### The merge process
+
+We use [`bors`](https://bors.tech/) to manage merging PR's in the the `EnsembleKalmanProcesses` repo.
+If you're a collaborator and have the necessary permissions, you can type
+`bors try` in a comment on a PR to have integration test suite run on that
+PR, or `bors r+` to try and merge the code.  Bors ensures that all integration tests
+for a given PR always pass before merging into `master`.
```
Comment (odunbar) [inline docs/src/contributing.md:159]: `main`

File: docs/src/contributing.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+Depending on the content changed in the PR, some CI checks that
+are not necessary will be skipped.  For example doc only changes
+do not require the unit tests to be run.
+
+### The merge process
+
+We use [`bors`](https://bors.tech/) to manage merging PR's in the the `EnsembleKalmanProcesses` repo.
+If you're a collaborator and have the necessary permissions, you can type
+`bors try` in a comment on a PR to have integration test suite run on that
+PR, or `bors r+` to try and merge the code.  Bors ensures that all integration tests
```
Comment (odunbar) [inline docs/src/contributing.md:158]: "The integration tests currently runs example cases in `examples/`. Any breaking changes will need to also update the `examples/`, else `bors` will fail "

**Review by odunbar — COMMENTED** [review]
LGTM after the small changes.

Thanks! I think this will improve my own contributions to the code as well!

File: docs/src/contributing.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+- When you start working on a new feature branch, make sure you start from
+  main by running: `git checkout main` and `git pull`.
+
+- Create a new branch from main by using `git checkout -b <branchname>`.
+
+### Develop your feature
+
+Make sure you add tests for your code in `test/` and appropriate documentation in the code and/or
+in `docs/`. Make sure your feature follows the formatting guidelines by running
```
[the remark this answers is not in the document]
Comment (ilopezgp) [reply docs/src/contributing.md:55]: I don't think they can. However, we should not rely on the examples as integration tests. So if this becomes an issue we should increase the test coverage.

**Pull request merged**